modules/swagger-codegen/src/main/resources/aspnetcore/Startup.mustache

```diff
@@ -61,6 +61,7 @@ namespace {{packageName}}
                         Description = "{{#appName}}{{{appName}}}{{/appName}}{{^appName}}{{packageName}}{{/appName}} (ASP.NET Core 2.0)",
                         Contact = new Contact()
                         {
+                           Name = "{{#infoName}}{{{infoName}}}{{/infoName}}",
```

Comment:
@knom What about defaulting to "Swagger Codegen Contributors" if `infoName` (name in contact) is not defined?

The goal is to do a bit of growth hacking for Swagger Codegen.

Same for `infoUrl`: defaulting to "https://github.com/swagger-api/swagger-codegen" if `infoURL` is not define.

Reply:
Sure thing!
Can you edit the commit?
I shutdown my build VM..

Reply:
Nice idea!
Let me update..